Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,25 @@
 import React from 'react';
 import './Button.css';
+
+
+const NumberButton = props => {
+    return (        
+            <button className='buttonStyle'>
+                <div className='clearDiv'>{props.eachNumber.textClear}</div>
+                <div className='numberDiv'>
+                    <div className='anumberDiv'>{props.eachNumber.textSeven}</div>
```

Comment:
You may have been able to map over these items and save yourself some code. Could be a refactor for practice later.